src/software/pipeline/main_featureMatching.cpp

```diff
@@ -107,7 +107,6 @@ int main(int argc, char **argv)
   size_t numMatchesToKeep = 0;
   bool useGridSort = true;
   bool exportDebugFiles = false;
-  std::string fileExtension = "bin";
```

Comment:
i'd rather keep the variable and declare it const
```c++
const std::string fileExtension = "txt";
```
and use it later instead of explicitly put `"txt"`

Reply:
Yes, done in next commit.